### ISSUE TITLE
Misalignment of elements in account settings tab on varying screen width

### ISSUE THREAD

**Issue by neeraj3029** [fossasia/accounts.susi.ai#542]
**Actual Behaviour**

Misalignment of elements in account settings tab on varying screen width

**Expected Behaviour**
The elements should be arrranged properly.

**Steps to reproduce it**
Open account settings tab.

**Screenshots of the issue**

![screenshot 2018-10-21 at 12 18 12 pm](https://user-images.githubusercontent.com/31539812/47264066-389eab00-d52c-11e8-88d8-6245fcf390b7.png)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/31539812/47264091-d5614880-d52c-11e8-86cc-22b2a2306314.gif)
**Would you like to work on the issue?**
Yes

**Comment by prateek76** [comment]
@neeraj3029 are you still working on this issue

**Comment by neeraj3029** [comment]
@prateek76 Yes I will make a PR on this after a while and please do make PR without consent.

**Comment by prateek76** [comment]
> @prateek76 Yes I will make a PR on this after a while and please do make PR without consent.

Ok, so can i make pr in this issue?

**Comment by neeraj3029** [comment]
No, I will work on this.

**Comment by samagragupta** [comment]
@prateek76 you should not make PR by going against the consent of the user who has created the issue unless some mentors have assigned you.

**Comment by prateek76** [comment]
@neeraj3029 how much time will you take please tell, i will see it after that. thanks

**Comment by samagragupta** [comment]
@prateek76 Have a look : https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

**Comment by prateek76** [comment]
> @prateek76 you should not make PR by going against the consent of the user who has created the issue unless some mentors have assigned you.

ok thanks, i am new to opensource

**Comment by neeraj3029** [comment]
@prateek76 I will return to this issue once I solve all the other important pending issues is SUSI, which may take a while. I will mention you when I find a relevant issue you can work upon. Please wait till then.

**Comment by prateek76** [comment]
> @prateek76 I will return to this issue once I solve all the other important pending issues is SUSI, which may take a while. I will mention you when I find a relevant issue you can work upon. Please wait till then.

thanks, i am working on that last issue as i found the fix

**Comment by neeraj3029** [comment]
@prateek76  You mean #585 ?

**Comment by prateek76** [comment]
> @prateek76 You mean #585 ?

yes
